src/main/java/binaries/josue/BinaryArray.java

```diff
@@ -2,22 +2,28 @@
 
 public class BinaryArray {
 
-    /**
-     * Array of 1s and 0s expected
-     */ 
-    private int[] binaries;
+	/**
```

Comment:
Please use 4 spaces instead of tabs. This should be configurable from your editor settings or IDE